server/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -633,21 +629,34 @@ public Iterator<? extends ToXContent> toXContentChunked(ToXContent.Params outerP
                 (builder, params) -> builder.endObject()
             ),
 
-            // transportVersions
+            // transportVersions - redundant with the nodes_versions section but has to stay for backwards compatibility
```

Comment:
This was only introduced in ?8.8, I very much doubt anyone is using it yet...

Reply:
Me too, and `GET _cluster/state` is technically a debug API and the [docs say it is unstable](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-state.html). I'm +1 to remove it, but we ought to run that change past the breaking changes committee first.

Reply:
I opened #99223 and a corresponding proposal to the breaking changes committee.